google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -162,13 +162,20 @@ appengine.deployment.deployallconfigs.checkbox.label=Update dos, dispatch, cron,
 appengine.deployment.flexible.nogaesupport=No modules with Google App Engine flexible framework support found.
 appengine.deployment.error.cancelled=Deployment process was cancelled.
 appengine.deployment.staging.artifact.as=Staging artifact as: {0}
+appengine.deployment.status.preparing.sdk=Waiting for Google Cloud SDK installation to complete...
+appengine.deployment.error.sdk.still.installing=Google Cloud SDK with App Engine Java needs to be completely installed to perform this action. Please wait for installation to complete.
+appengine.deployment.error.sdk.not.available=Google Cloud SDK is not available.
+appengine.deployment.error.sdk.invalid=Google Cloud SDK is invalid.
+appengine.deployment.error.sdk.settings.action=Open Google Cloud SDK Settings...
+
 appengine.converter.description=Convert projects created by the legacy App Engine plugin to be compatible with the new plugin
 appengine.staged.artifact.name.converter.description=Existing App Engine flexible deployment run configurations will have their staged artifact name explicitly set to target.jar/war for interoperability with existing Dockerfiles.
 appengine.promoteinfo.url=https://console.cloud.google.com/appengine/versions
 appengine.run.server.name=Google App Engine Standard Local Server
 appengine.run.server.artifact.missing=Artifact to deploy isn't specified. If no artifacts are available, ensure the following: <br /> 1) An 'exploded-war' artifact is configured for your desired deployment. <br />2) The App Engine standard facet is applied to the corresponding module(s) you wish to run. You can use the menu action under 'Tools -> Google Cloud Tools -> Add App Engine support -> Google App Engine Standard'.
-appengine.run.server.sdk.misconfigured.panel.message=The Cloud SDK is misconfigured. To fix, click the 'Configure' button next to the application server.
-appengine.run.server.sdk.misconfigured.message=The Cloud SDK is misconfigured. To fix, reconfigure the application server.
+appengine.run.server.sdk.misconfigured.panel.message=The Cloud SDK is misconfigured. To fix, use Google -> Cloud SDK settings.
+appengine.run.server.sdk.misconfigured.message=The Cloud SDK is misconfigured. To fix, use Google -> Cloud SDK settings.
```

Comment:
this and the one above look to be the same message?

Reply:
Yep. First one is shown for local server config panel, second one for error when server run failed. Considering that server will have additional error message with link to settings from common precondition check, I think we can remove the second part.

Reply:
was this comment addressed? If I'm not mistaken it looks like there are still duplicated messages.

Reply:
that is, 
`appengine.run.server.sdk.misconfigured.panel.message` and
`appengine.run.server.sdk.misconfigured.message `

Reply:
Correct, they are the same for now, but I plan to expand the message shown after actual local run (`appengine.run.server.sdk.misconfigured.message`) so it could include additional information and possibly "local run" specific wording (part of #1968). Sorry for the confusion

Reply:
got it. sgtm